docs/developers.md

```diff
@@ -0,0 +1,39 @@
+# Linting
+
+To ensure consistent formatting across developres, we include congifurations for Prettier (code formatting), ESLint (JS), and StyleLint (SCSS) to apply consistent styles.
+
+Inside of the `website` folder, you can run `yarn eslint` to list issues, or `yarn eslint:fix` to automatically fix what is possible.
+
+Similarly to lint stylesheets, you can run `yarn stylelint` or `yarn stylelint:fix`.
+
+To lint as you go, we reccomend configuring your editor to use this config and apply fixes on save, and display warnings. For VSCode, you can do that by installing the eslint, prettier and stylelint plugins.
+
+To ensure we're using our project specified configuration, you'll want to open settings via `Preferences: Open Settings (JSON)`, and add the following rules:
+
+```json
+// Global default is to use built-in prettier
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  // For our JS and SCSS Files, we want project defined versions
+  "[javascript]": {
+    "editor.formatOnSave": false
+  },
+  "[scss]": {
+    "editor.formatOnSave": false
+  },
+  "files.associations": {
+    "*.js": "javascript",
+    "*.scss": "scss"
+  },
+  // Turn on auto-fixing for project defined linters.
+  "editor.codeActionsOnSave": {
```

Comment:
Talked about this offline, and clarified that auto fix only applies to things it easily can.

Since the editor settings will be optional, we need a way to run the linting via hooks, therefore need to switch to https://github.com/okonet/lint-staged instead of pretty quick.